packages/@coorpacademy-player-services/src/test/fixtures/data/progressions.js

```diff
@@ -0,0 +1,566 @@
+        payload: {
+          content: {
+            type: 'slide',
+            ref: 'sli_41GXTiG8g'
+          },
+          answer: [],
+          isCorrect: true,
+          godMode: false,
+          instructions: null,
+          nextContent: {
+            type: 'slide',
+            ref: 'sli_41GXTiG8X'
+          }
+        }
+      }
+    ]
+  },
+  {
+    _id: 'livesDisabled',
+    engine: {
+      ref: 'learner',
+      version: '1'
+    },
+    engineOptions: {
+      livesDisabled: true
+    },
+    content: {
+      ref: '1.A',
+      type: 'level'
+    },
+    actions: [
+      {
+        type: 'move',
+        payload: {
+          nextContent: {
+            type: 'slide',
+            ref: 'sli_41GXTiG8X',
+            instructions: null
+          }
+        }
+      }
+    ]
+  },
+  {
+    _id: 'extraLife',
+    engine: {
+      ref: 'microlearning',
+      version: '1'
+    },
+    engineOptions: {},
+    content: {
+      ref: '5.C7',
+      type: 'chapter'
+    },
+    actions: [
+      {
+        type: 'move',
+        payload: {
+          nextContent: {
+            type: 'slide',
+            ref: 'sli_VkcAWR-ZQ',
+            instructions: null
+          }
+        }
+      },
+      {
+        type: 'answer',
+        payload: {
+          content: {
+            type: 'slide',
+            ref: 'sli_VkcAWR-ZQ'
+          },
+          answer: [],
+          isCorrect: false,
+          godMode: false,
+          instructions: null,
+          nextContent: {
+            type: 'node',
+            ref: 'extraLife'
+          }
+        }
+      }
+    ]
+  },
+  {
+    _id: 'adaptive',
+    engine: {
+      ref: 'microlearning',
+      version: '1'
+    },
+    engineOptions: {},
+    content: {
+      ref: 'cha_N19MiQrYG',
+      type: 'chapter'
+    },
+    actions: [
+      {
+        type: 'move',
+        payload: {
+          nextContent: {
+            type: 'slide',
+            ref: 'sli_4yHKRmBtz',
+            instructions: null
+          }
+        }
+      }
+    ]
+  },
+  {
+    _id: 'partialAdaptive',
+    engine: {
+      ref: 'learner',
+      version: '1'
+    },
+    engineOptions: {},
+    content: {
+      ref: 'mod_E15Bh1hDM',
+      type: 'level',
+      version: '1'
+    },
+    actions: [
+      {
+        type: 'move',
+        payload: {
+          nextContent: {
+            type: 'slide',
+            ref: 'sli_4JE0EO73M',
+            instructions: null
+          },
+          instructions: null
+        }
+      },
+      {
+        type: 'answer',
+        payload: {
+          answer: ['objectif', 'expertises'],
+          content: {
+            type: 'slide',
+            ref: 'sli_4JE0EO73M'
+          },
+          godMode: false,
+          nextContent: {
+            type: 'slide',
+            ref: 'sli_NJenMOm2f'
+          },
+          isCorrect: false,
+          instructions: null
+        }
+      }
+    ]
+  }
+];
+
+export default progressions;
```

Comment:
that's too bad that you're not importing other fixtures to compose this object :(